### ISSUE TITLE
Fail to hexo generator, if it is a org-mode file to be createded

### ISSUE THREAD

**Issue by yssource** [CodeFalling/hexo-renderer-org#28]
# Development Environment

-   fc24.x86\_64
-   GNU Emacs 25.1.1
-   "hexo-renderer-org": "^0.2.4",

# Reproduce

creating a new post with file name "test2.org"

    M-x blog-admin-start w

# Result

    XXX==Error Here==>/var/www/org/blog/source/_drafts/test2.org:

# Test on travis-ci.org
![screenshot from 2016-10-21 18-09-50](https://cloud.githubusercontent.com/assets/3756334/19595241/7757a6ea-97bb-11e6-846b-dd62ea283aa9.png)

**Comment by yssource** [comment]
unable to pass any test for org-mode files on [https://travis-ci.org](https://travis-ci.org)
```
Symbol's function definition is void: org-html-export-as-html

(node:3594) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: Cannot read property 'parent' of null
```

**Comment by yssource** [comment]
My problem was solved, after upgrading to the latest version 25.1, since Emacs on traivs-cli.org is out of date. 
Thanks for your extension.
